lib/web_ui/skwasm/BUILD.gn

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Hmm, that's a bit orthogonal to this change. I don't really disagree with that idea, but I don't actually know the original reasoning behind why `web_sdk` is in its own separate directory. Maybe @yjbanov knows?

Also, I'd have to check, but I think there is some code downstream in the recipes that relies on the location of `web_sdk` so it might not be as simple as just moving it to a different directory.

Reply:
I agree it's orthogonal. It's a bit weird that some of our build files live in `web_sdk` at the root of the repo, while others live inside `web_ui`.

Reply:
Yeah I'm with you on that.